scripts/lib/CIME/SystemTests/erp.py

```diff
@@ -57,6 +57,10 @@ def _case_two_setup(self):
         self._case.set_value("CONTINUE_RUN", True)
         self._case.set_value("REST_OPTION","never")
 
+        # May need to change EXEROOT since separate_rundirs is False and separate_builds is True
```

Comment:
I feel like this code belongs somewhere in system_tests_compare_two rather than in erp.py, so that other future tests that need it don't need to duplicate this. i.e., somewhere in system_tests_compare_two, if you have separate_rundirs=False and separate_builds=True, then it would set case2's EXEROOT like this. (Then you could possibly remove the error check that you just added about this in system_tests_compare_two.)